samples/tab-meeting-transcript-recording/csharp/MeetingTranscriptRecording/ClientApp/src/components/auth-start.jsx

```diff
@@ -0,0 +1,50 @@
+// <copyright file="auth-start.jsx" company="Microsoft Corporation">
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+// </copyright>
+
+import { useEffect } from "react";
+import * as microsoftTeams from "@microsoft/teams-js";
+import * as msal from "@azure/msal-browser";
+
+const AuthStart = props => {
+
+    useEffect(() => {
+
+        async function AuthenticationStart() {
+            microsoftTeams.app.initialize();
+            const context = await microsoftTeams.app.getContext();
+            var scope = "User.Read email openid profile offline_access";
```

Comment:
I don't see any permission related to OnlineMeeting, try following.
1. Remove the admin consent for this app from Contoso tenant: 
![image](https://github.com/OfficeDev/Microsoft-Teams-Samples/assets/31851992/d7b6ffe0-8390-4cdb-8b6f-b411692417d1)
2. Add these permissions directly in auth-shart.jsx file.
3. Test the auth flow and check if we are getting token with correct permission.

Reply:
Fixed